airflow/dags/transform_warehouse_sunday_select_rt/METADATA.yml

```diff
@@ -0,0 +1,18 @@
+description: "Builds and tests RT data in the warehouse every sunday, this runs weekly on Sundays to compliment transform_warehouse_sunday_full_refresh_exclude_rt"
+schedule_interval: "0 12 * * 0"
+tags:
+  - all_gusty_features
+default_args:
+    owner: airflow
+    depends_on_past: False
+    start_date: !days_ago 1
+    email:
+      - "andrew.v@jarv.us"
+      - "jameelah.y@jarv.us"
```

Comment:
**thought (non-blocking):** I wonder whether there are any Caltrans addresses we can start changing these to. Maybe a conversation for the next 12L check-in?

Reply:
I just changed this to Hunter and Evan, which is how some of our other, newer dags are configured